pages/common/pulumi-new.md

```diff
@@ -21,4 +21,4 @@
 
 - Use the specified secrets provider with the <pulumi.com> backend:
 
-`pulumi new --secrets-provider={{passphrase}}`
+`pulumi new --secrets-provider {{passphrase}}`
```

Comment:
```suggestion
`pulumi new --secrets-provider={{passphrase}}`
```
The `=` is required according to the official docs: https://www.pulumi.com/docs/iac/cli/commands/pulumi_new/

Reply:
It isn't
`--secrets-provider string     The type of the provider that should be used to encrypt and decrypt secrets (possible choices: default, passphrase, awskms, azurekeyvault, gcpkms, hashivault) (default "default")`